index.js

```diff
@@ -1123,6 +1123,48 @@ class Encore {
         return this;
     }
 
+    /**
+     * If enable, browser sync will start to provide live reload functionality
+     *
+     * https://www.browsersync.io/
+     *
+     * Call it to start a proxy redirecting requests to `backendUrl` while
```

Comment:
You're talking about `backendUrl` here and a bit below but no actual option is called like that, so it's a bit confusing. Did you mean `proxy`?